app/uploaders/file_upload.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+class FileUpload
+  attr_accessor :options
+  class << self
+    def uploader(uploader = nil)
```

Comment:
Thinking about it, I like the class_attributes better because it's clear you're setting _one_ thing, unlike `run` in the workflows where you keep adding things.